transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -209,12 +221,13 @@ private void registerService(BundleContext bundleContext) {
 
     private List<String> getSupportedMimeTypes() {
         SortedSet<MediaType> mediaTypes = MediaTypeRegistry.getDefaultRegistry().getTypes();
-        List<String> mimeTypes = new ArrayList<String>(mediaTypes.size());
+        List<String> mimeTypes = new ArrayList<>(mediaTypes.size());
 
         for (MediaType mediaType : mediaTypes) {
             String mimeType = mediaType.getType() + "/" + mediaType.getSubtype();
             mimeTypes.add(mimeType);
         }
+        mimeTypes.add("image/jp2");
```

Comment:
According to @stustison, that would require upgrading some other stuff as well, so it's not as convenient as it should be. Otherwise that would probably be easier.